submissions/cynthiaNwakaeme/CSS/contact.css

```diff
@@ -0,0 +1,277 @@
+    width: 35px;
+    text-align: center;
+    word-spacing: 2px;
+    height: 35px;
+}
+        /* MAIN FRAME */
+main{
+    float: left;
+    text-align: center;
+    width: 50%;
+    padding-top: 3%;
+    
+}
+
+main h3, aside h3{
+    color: rgb(58, 57, 57);
+}
+main ul li{
+    display: inline;
+    padding: 2%;
+}
+
+       /* ASIDE FRAME */
+aside{
+    float: left;
+    width: 43%;
+    margin-left: 5%;
+}
+.enquiry-details{
+    overflow: hidden;
+    padding: 20px;
+    width: 90%;
+    border:1px solid rgb(129, 129, 129);
+    border-radius: 3%;
+    margin-bottom: 10px;
+}
+.enquiry-details img{
+    padding: 0 45%;
+}
+.enquiry-details h3{
+    text-align: center;
+    width: 95%; 
+}
+.enquiry-details input{
+    border-top-style: hidden;
+    border-right-style: hidden;
+    border-left-style: hidden;
+    border-bottom: 1px groove rgb(129, 129, 129);
+    width: 95%;
+    padding-top: 5%;
+}
+.enquiry-details input:focus{
+    outline: none;
+
+  }
+.enquiry-details input::placeholder{
+    color:rgb(129, 129, 129) ;
+    font-size: 15px;
+}
+.enquiry-details label{
+    padding-top: 5%;
+    color:rgb(129, 129, 129) ;
+}
+.enquiry-details textarea{
+    resize: none;
+    width: 95%; 
+}
+.enquiry-details button{
+    float: right;
+    width: 25%;
+    height: 40px;
+    background-color: #9d426b;
+    color: #fff;
+    margin: 2% 4% 0;
+    padding: 2%;
+    border: none;
+    font-size: 18px;
+}
+
+/* THE FOOTER */
+footer{
+    background-color: #9d426b;
+    padding:20px 0 0 50px;
+    color: white;
+    overflow: hidden;
+    clear: both;
+}
+.footer-box,
+.footer-box-2,
+.footer-box-3{
+    float: left;
+    width: 33%;
+}
+.footer-box ul li,
+.footer-box-3 ul li
+{
+    list-style: none;
+    margin-bottom: 8px;
+}
+.footer-box ul li a{
+    text-decoration: none;
+    display: inline;
+    color: white;  
+}
+.footer-box-2 button[type="submit"]{
+    height: 30px;
+    padding: 0 5%;
+    font-size: 12px;
+    color: #9d426b;
+    font-weight: bold;
+}
+.footer-box-2 input[type="email"]{
+    height: 20px;
+    padding: 3px 18px 3px ;
+    font-size: 11px; 
+}
+.footer-box-3 ul li{
+    display: inline-flex;
+    padding:10px;
+    
+}
+.footer-box-3 ul li img{
+    width: 35px;
+    height: 30px;
+}
+footer p{
+    text-align: center;
+    display: block;
+}
+.footer-box,
+.footer-box-2,
+.footer-box-3{
+    float: left;
+    width: 33%;
+}
+.footer-box ul li,
+.footer-box-3 ul li
+{
+    list-style: none;
+    margin-bottom: 8px;
+}
+.footer-box ul li a{
+    text-decoration: none;
+    display: inline;
+    color: white;  
+}
+.footer-box-2 button[type="submit"]{
+    height: 30px;
+    padding: 0 5%;
+    font-size: 12px;
+    color: #9d426b;
+    font-weight: bold;
+}
+.footer-box-2 input[type="email"]{
+    height: 20px;
+    padding: 3px 18px 3px ;
+    font-size: 11px; 
+}
+.footer-box-3 ul li{
+    display: inline-flex;
+    padding:10px ;
+}
+.footer-box-3 ul li img{
+    width: 35px;
+}
+footer p{
```

Comment:
You have declared this previously.

Reply:
@Cynthia-Berry you have the same thing in the contents, that's repetition.

Reply:
Hello @ToluAkin kindly specify for which tags cause my styling on `li` tag is different from the `li img` styling.